misc-scripts/xref_mapping/XrefParser/RefSeqCoordinateParser.pm

```diff
@@ -369,7 +387,7 @@ sub run_script {
 # If a best match was defined for the refseq transcript, store it as direct xref for ensembl transcript
         if ($best_id) {
           my ($acc, $version) = split(/\./, $id);
-	  $version =~ s/\D//g if $version;
+	        $version =~ s/\D//g if $version;
```

Comment:
cosmetic changes should be avoided, and if absolutely necessary, submitted as a separate commit to avoid masking the actual code changes